dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -552,7 +556,20 @@ CFileDefExt::InitFileAttr(HWND hwndDlg)
             FileSize.u.LowPart = FileInfo.nFileSizeLow;
             FileSize.u.HighPart = FileInfo.nFileSizeHigh;
             if (SH_FormatFileSizeWithBytes(&FileSize, wszBuf, _countof(wszBuf)))
+            {    
                 SetDlgItemTextW(hwndDlg, 14011, wszBuf);
+                // Calculate size on disc
+                if( GetVolumePathName(m_wszPath, lpszVolumePathName, sizeof(lpszVolumePathName) / sizeof(TCHAR)) ) {
```

Comment:
Ionescu's school of parenthesing? (and in other places)
```suggestion
                if (GetVolumePathName(m_wszPath, lpszVolumePathName, sizeof(lpszVolumePathName) / sizeof(TCHAR)))
                {
```

Reply:
the school of parenthesizing is when you add many more parentheses than needed.

Reply:
I thought it's about putting spaces into the inner condition too :)

Reply:
fixed